libraries/Microsoft.Bot.Builder.AI.LUIS-preview/LuisApplication.cs

```diff
@@ -0,0 +1,115 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using System.Linq;
+using System.Web;
+using Microsoft.Bot.Configuration;
+
+namespace Microsoft.Bot.Builder.AI.LuisPreview
+{
+    /// <summary>
+    /// Data describing a LUIS application.
+    /// </summary>
+    public class LuisApplication
+    {
+        public LuisApplication()
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="LuisApplication"/> class.
+        /// </summary>
+        /// <param name="applicationId">LUIS application ID.</param>
+        /// <param name="endpointKey">LUIS subscription or endpoint key.</param>
+        /// <param name="endpoint">LUIS endpoint to use like https://westus.api.cognitive.microsoft.com.</param>
+        public LuisApplication(string applicationId, string endpointKey, string endpoint)
+            : this((applicationId, endpointKey, endpoint))
```

Comment:
Why "((" on all of these?

Reply:
BEcause it is an anonymous tuple...

Reply:
I removed it from the solution--I'll remove the files as well.